client/src/components/LoginSession/tests/LoginSession-test.js

```diff
@@ -1,19 +1,16 @@
-/* global jest, jasmine, describe, it, expect, beforeEach, Event */
+/* global jest, describe, it, expect, beforeEach, Event */
 
 import React from 'react';
 import LoginSession from '../LoginSession';
 import Enzyme, { shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-import MockDate from 'mockdate';
-
-jest.mock('@silverstripe/reactstrap-confirm', () => () => Promise.resolve(true));
```

Comment:
This was causing a failure (couldn't resolve the library) but simply isn't necessary anymore in any case. Tests pass without it.